exercises/practice/killer-sudoku-helper/.meta/example.py

```diff
@@ -0,0 +1,13 @@
+import itertools
+
+def combinations(target, size, exclude):
+    result = []
+    if size == 1:
+        return [[target]]
+    else:
+        possible = [index for index in range(1, int((target ** 2 / size) ** 0.6)) if index not in exclude]
+        for index in range(len(possible), 0, -1):
+            for seq in itertools.combinations(possible, index):
+                if sum(seq) == target and len(seq) == size:
+                    result.append(list(seq))
+    return result
```

Comment:
```suggestion
import itertools

def combinations(target, size, exclude):
    result = []
    possible = [index for index in
                range(1, int((target ** 2 /size) ** 0.6))
                if index not in exclude]

    if size == 1:
        return [[target]]
    else:
        for index in range(len(possible), 0, -1):
            for seq in itertools.combinations(possible, i):
                if sum(seq) == target and len(seq) == size:
                    result.append(list(seq))
    return result
```

Reply:
Juuust to clean up the formatting a tad.  Otherwise, looks great.

Reply:
I didn't look through it that clearly
But you changed from index to i

Reply:
Otherwise it is a lot better